disk/test/com/treode/disk/DiskTestTools.scala

```diff
@@ -40,6 +41,15 @@ private object DiskTestTools {
 
   val sysid = SystemId (0, 0)
 
+  def assertEqNotification (expected: Message*) (actual: Notification) {
+    if (actual.isEmpty) {
+      assert (expected.length == 0)
+    } else {
+      assert (expected == actual.messages)
+      assert (expected.length == actual.messages.length)
+    }
+  }
+
```

Comment:
Awesome. You're almost there. All that's left is to simplify this. `expected == actual.messages` will do. There's no need to special case empty. After checking that the sequences are equal, it's redundant to check that their lengths are equals.